plugin/src/ast_helpers.rs

```diff
@@ -664,6 +666,110 @@ pub fn is_tagged_template_expression(node: Node) -> bool {
     node.kind() == CallExpression && node.field("arguments").kind() == TemplateString
 }
 
+pub fn get_template_string_chunks<'a>(
```

Comment:
Per https://github.com/tree-sitter/tree-sitter-javascript/issues/275 template literal "string chunks" aren't currently exposed as individual nodes so this is an attempt to be able to iterate over their string contents + positions within the parent `template_string` node